src/blockdata/transaction.rs

```diff
@@ -1733,6 +1733,23 @@ mod tests {
             _ => panic!("Wrong error type"),
         }
     }
+
+    // Used to verify that parts of a transaction pretty print.
+    // `cargo test display_transaction --features=serde -- --nocapture`
+    #[cfg(feature = "serde")]
+    #[test]
+    fn serde_display_transaction() {
+        let tx_bytes = Vec::from_hex(
+            "02000000000101595895ea20179de87052b4046dfe6fd515860505d6511a9004cf12a1f93cac7c01000000\
+            00ffffffff01deb807000000000017a9140f3444e271620c736808aa7b33e370bd87cb5a078702483045022\
+            100fb60dad8df4af2841adc0346638c16d0b8035f5e3f3753b88db122e70c79f9370220756e6633b17fd271\
+            0e626347d28d60b0a2d6cbb41de51740644b9fb3ba7751040121028fa937ca8cba2197a37c007176ed89410\
+            55d3bcb8627d085e94553e62f057dcc00000000"
+        ).unwrap();
+        let tx: Transaction = deserialize(&tx_bytes).unwrap();
+        let ser = serde_json::to_string_pretty(&tx).unwrap();
+        println!("{}", ser);
```

Comment:
I think we should use `assert_eq`. I know changes to serde would break this but seems less bad than manually checking with funny flags.

Reply:
I decided to remove the test all together. I put it in the PR description in case anyone wants to verify that this PR works as advertised.